### PR TITLE
TGP 2066 : record lock advice status

### PR DESCRIPTION
**Pull request by vinoocs**

### CODE REVIEW

**Review by Ewan-James-Donovan — APPROVED** [review]

File: app/controllers/SingleRecordController.scala [hunk 1]
```diff
@@ -52,6 +52,14 @@ class SingleRecordController @Inject() (
     (identify andThen profileAuth andThen getData andThen requireData).async { implicit request =>
       for {
         record                             <- goodsRecordConnector.getRecord(request.eori, recordId)
+        recordIsLocked                      = record.adviceStatus match {
```
Comment (ChrisCookOC) [inline app/controllers/SingleRecordController.scala:55]: Not necessary if you are happy, but if you want to, you could add this as a method on the record itself, and then just call that here

File: test/controllers/SingleRecordControllerSpec.scala [hunk 2]
```diff
@@ -239,5 +248,298 @@ class SingleRecordControllerSpec extends SpecBase with MockitoSugar {
       }
     }
 
+    "TraderReferenceSummary.row" - {
```
Comment (ChrisCookOC) [inline test/controllers/SingleRecordControllerSpec.scala:251]: These tests are in the wrong place?

**Review by ChrisCookOC — APPROVED** [review]

File: test/controllers/SingleRecordControllerSpec.scala (hunk 2 above)
Comment (vinoocs) [reply test/controllers/SingleRecordControllerSpec.scala:251]: No, they are in the right place. As we display these rows on single record page, added these tests here.

**Review by mycp98 — APPROVED** [review]

File: app/controllers/SingleRecordController.scala (hunk 1 above)
Comment (vinoocs) [reply app/controllers/SingleRecordController.scala:55]: can do this as improvement in the other PR.

**Pull request merged**